src/components/content/order/formElements/TextInput.tsx

```diff
@@ -74,6 +112,24 @@ export function TextInput({ item }: { item: DeployParam }): React.JSX.Element {
                             }
                             onChange={getStringEventHandler}
                         />
+                    ) : item.autoFill.isAllowCreate ? (
+                        <AutoComplete

```

Comment:
this is for sure not very clear for the customer. We need something where a user explicitly selects that "he wants to create a new one" and then gives a new value.